vmdb/app/models/ems_refresh/parsers/foreman.rb

```diff
@@ -130,18 +132,22 @@ def configuration_profile_inv_to_hashes(recs, osfs, media, ptables, locations, o
         end
       end
 
-      def configured_system_inv_to_hashes(recs, profiles, operatingsystems, locations, organizations)
+      def configured_system_inv_to_hashes(recs, profiles, operatingsystems, media, ptables, locations, organizations)
         recs.collect do |cs|
           {
-            :type                          => "ConfiguredSystemForeman",
-            :manager_ref                   => cs["id"].to_s,
-            :hostname                      => cs["name"],
-            :configuration_profile         => id_lookup(profiles, cs, "hostgroup_id"),
-            :operating_system_flavor_id    => id_lookup(operatingsystems, cs, "operatingsystem_id"),
-            :last_checkin                  => cs["last_compile"],
-            :build_state                   => cs["build"] ? "pending" : nil,
-            :configuration_location_id     => id_lookup(locations, cs, "location_id"),
-            :configuration_organization_id => id_lookup(organizations, cs, "organization_id"),
+            :type                           => "ConfiguredSystemForeman",
+            :manager_ref                    => cs["id"].to_s,
+            :hostname                       => cs["name"],
+            :configuration_profile          => id_lookup(profiles, cs, "hostgroup_id"),
+            :operating_system_flavor_id     => id_lookup(operatingsystems, cs, "operatingsystem_id"),
+            :customization_script_medium_id => id_lookup(media, cs, "medium_id"),
+            :customization_script_ptable_id => id_lookup(ptables, cs, "ptable_id"),
+            :last_checkin                   => cs["last_compile"],
+            :build_state                    => cs["build"] ? "pending" : nil,
+            :ipaddress                      => cs["ip"],
+            :mac_address                    => cs["mac"],
```

Comment:
tracking down the values for this.
Foreman team gave me a grep phrase to track down what the values mean.
(`status.applied='false'`). Then I can write the docs and people would know what it means.